trendmicrovisionone_connector.py

```diff
@@ -835,850 +733,1438 @@ def _handle_remove_from_blocklist(self, param):
+            response = client.add_to_exception_list(
+                pytmv1.ObjectTask(
+                    object_type=i["object_type"],  # type: ignore
+                    object_value=i["object_value"],
+                    description=i.get("description", "Add To Exception List."),
+                )
+            )
+            if "error" in response.result_code.lower():
+                raise RuntimeError(
+                    f"Error while adding to exception list: {response.errors}"
+                )
+            assert response.response is not None
+            items = response.response.dict().get("items", [])[0]
+            items["task_id"] = (
+                "None" if items.get("task_id") is None else items["task_id"]
+            )
+            multi_resp.append(items)
+
+        # Get total exception list count
+        total_exception_count = self.exception_list_count()
+        # Add the response into the data section
+        action_result.add_data(
+            {"multi_response": multi_resp, "total_count": total_exception_count}
+        )
+
+        # Return success
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_delete_from_exception(self, param):
+        """
+        Deletes the specified objects from the Exception List.
+        Args:
+            block_objects(List[Dict[str, str]]): List of objects containing type, value and optional description.
+        Returns:
+            multi_resp(Dict[str, List]): Object containing task_id and HTTP status code.
+        """
+        self.save_progress(
+            "In action handler for: {0}".format(self.get_action_identifier())
+        )
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Required Params
+        block_objects: List[Dict[str, str]] = json.loads(param["block_objects"])
+
+        # Initialize Pytmv1
+        client = self._get_client()
+
+        multi_resp: List[MsData] = []
+
+        # Choose Enum
+        for i in block_objects:
+            obj_type = i["object_type"].upper()
+            if obj_type in ObjectType.__members__:
+                i["object_type"] = ObjectType[obj_type]
+            else:
+                raise RuntimeError(f"Please check object type: {i['object_type']}")
+
+        # Make rest call
+        for i in block_objects:
+            response = client.remove_from_exception_list(
+                pytmv1.ObjectTask(
+                    object_type=i["object_type"], object_value=i["object_value"]  # type: ignore
+                )
+            )
+            if "error" in response.result_code.lower():
+                raise RuntimeError(
+                    f"Error while removing from exception list: {response.errors}"
+                )
+            assert response.response is not None
+            items = response.response.dict().get("items", [])[0]
```

Comment:
Similar to above, avoid calling .dict() here unnecessarily.

Reply:
done.